docs/decisions/arch-diagrams/mobile-create-order.mmd

```diff
@@ -0,0 +1,72 @@
+sequenceDiagram
+    participant User
+    participant MobileApp
+    participant Backend
+    participant TokenChecker
```

Comment:
What is a TokenChecker here? A separate application? Can this be an operation from the CoFE BE to Google/Apple?